doc/nrf/drivers/wifi/nrf70_native.rst

```diff
@@ -82,7 +82,7 @@ The nRF Wi-Fi driver has the following configuration options:
 Kconfig configuration
 =====================
 
-.. options-from-kconfig:: /../../../../../zephyr/drivers/wifi/nrfwifi/Kconfig.nrfwifi
+.. options-from-kconfig:: /../../../../../zephyr/drivers/wifi/nrf_wifi/Kconfig.nrf_wifi
```

Comment:
The doc build error seems to be due to the folder path which is "drivers/wifi/nrf_wifi/Kconfig.nrfwifi" not "drivers/wifi/nrf_wifi/**Kconfig.nrf_wifi**" as per https://github.com/nrfconnect/sdk-zephyr/pull/2316/files#diff-36828457087e584a38d19121aafd768954fa2b78f98aef837961de7d5fa25402.

Reply:
yes,it's fixed locally thanks, didn't do another push to aggregated with any other fixes, so, was waiting for CI to complete.